client/disputes/index.tsx

```diff
@@ -74,7 +86,7 @@ const headers: DisputesTableHeader[] = [
 		required: true,
 	},
 	{
-		key: 'customer',
+		key: 'name',
```

Comment:
How about `customerName`?

Reply:
Nice suggestion. Following your advice I am also using `customerEmail` and `CustomerCountry`.

Reply:
Thank you :)